release-notes/EPM/I-DOCSIS/EPM_7.0.2_I-DOCSIS.md

```diff
@@ -0,0 +1,38 @@
+---
+uid: EPM_7.0.2_I-DOCSIS
+---
+
+# EPM 7.0.2 I-DOCSIS
+
+## New features
+
+#### Harmonic CableOS: New PTP page [ID_39310]
+
+A new PTP page has been added to the Harmonic CableOS connector. On this page, the following new parameters are available:
+
+- CCAP PTP Clock State
+- RPD Port PTP Status Table
+- RPD PTP Clock Status Table
+
+#### New map dashboards added [ID_39319]
+
+New dashboards titled *03. CPE MAP* have been added to the solution, which will show a map of the passive device information associated with the selected entity. These maps can be accessed from the Dashboard module or via a new dashboard link on the Overview page when the selected entity is opened from the Topology app.
+
+The new maps are available for the following levels:
+
+- Node segment
+- Node
+- Amplifier
+- Tap
+
+#### New Ping and Tracert buttons on CCAP and CM overview pages [ID_39334]
+
+On the overview page for a CCAP or CM, users can now click dedicated *Ping* and *Tracert* buttons to open a window with the outcome of the ping or traceroute operation for the relevant CCAP or CM. This will provide additional insight into the status and connectivity of the system's devices.
+
```

Comment:
Hello @miguel-slc, we implemented a specific DITT EPM I-DOCSIS installation package, but Rene did not approve of that solution. So, the deployment of DITT is going to be handled by us with the installation package that is going to be hidden in the catalog.

Reply:
Hi @EstebanSkyline, as discussed, the DITT EPM I-DOCSIS installation is not required in this case.
We could mention here that DITT solution is required to have this functionality available. If there is already documentation available for DITT in dataminer docs, we can include the corresponding link, otherwise we can mention that that solution is still not available

Reply:
As DITT is now available in the documentation, I've added a note that DITT must be deployed for this feature, with a link to the DITT section.